backend/inspirehep/records/api/mixins.py

```diff
@@ -13,7 +14,94 @@
 LOGGER = structlog.getLogger()
 
 
-class CitationMixin:
+class AuthorsPapersMapExtension:
```

Comment:
maybe rename it to ```PapersAuthorsExtension/Mixin```? or just lose the ```Map``` from the name

Reply:
:+1: